packages/nextjs/utils/scaffold-eth/fetchPriceFromUniswap.ts

```diff
@@ -1,13 +1,13 @@
 import { CurrencyAmount, Token } from "@uniswap/sdk-core";
 import { Pair, Route } from "@uniswap/v2-sdk";
 import { Address, createPublicClient, http, parseAbi } from "viem";
-import { mainnet } from "wagmi";
+import { mainnet } from "viem/chains";
 import scaffoldConfig from "~~/scaffold.config";
 import { ChainWithAttributes } from "~~/utils/scaffold-eth";
 
 const publicClient = createPublicClient({
   chain: mainnet,
-  transport: http(`${mainnet.rpcUrls.alchemy.http[0]}/${scaffoldConfig.alchemyApiKey}`),
+  transport: http(`https://eth-mainnet.g.alchemy.com/v2/${scaffoldConfig.alchemyApiKey}`),
```

Comment:
Should we use the `getAlchemyHttpUrl` util here?

Reply:
Ohh yeah ! Actually I was using it earlier but removed it here in https://github.com/scaffold-eth/scaffold-eth-2/pull/700/commits/f8ad6b3b5481c8c4c973a5f13e2eef6810f775a5 because of cyclic dependency, which was causing whole app to break :( 

But yup updated at https://github.com/scaffold-eth/scaffold-eth-2/pull/700/commits/7674eaed025e401d0d9b7382c9871dffa4b650c7 and used relative imports 

Thanks 🙌